### ISSUE TITLE
Add .travis.yml

### ISSUE THREAD

**Issue by finkiki** [NixOS/nixos-homepage#63]
Can you add a .travis.yml file ?

**Comment by davidak** [comment]
We need infrastructure to be able to translate it.

Also it will be a huge amount of work that we might not be able to complete with the current community size in reasonable time.

Maybe a tutorial or video in your language helps people to understand what Nix/NixOS is and how to start using it. But a little english is currently needed to use it i guess.

**Comment by wmertens** [comment]
Another issue is that nixpkgs uses English throughout, and thus all the documentation for nixos services and options is in English.

Translating that would mean moving the text out of nixpkgs, and updating it whenever an option changes. Lots of work and lots of opportunity for work being blocked pending translations :-/

**Comment by garbas** [comment]
I agree that nix community is small. It would be at least interesting to have this in mind when and maybe trying to organize the translations like rust-lang.org does it (https://github.com/rust-lang/www.rust-lang.org/blob/master/TRANSLATIONS.md)
